### ISSUE TITLE
remove-unreachable-code deletes useful code

### ISSUE THREAD

**Issue by esvit** [coderaiser/putout#17]
I faced with this issue on a real project, `putout` breaks the code when removing everything after the return statement.

Example
```js
function test() {
	functionAfterReturn();
	return console.info('test');

	function functionAfterReturn() {
		console.info('functionAfterReturn');
	}
}
test()
```

After putout
```js
'use strict';
function test() {
    functionAfterReturn(); //  <--- ReferenceError: functionAfterReturn is not defined
    return console.info('test');
}
test();
```

**Comment by coderaiser** [comment]
This is a very good example, I should think about it :).

**Comment by coderaiser** [comment]
Actually, what should be done is check if one of `next sibligns` is a `FunctionDeclaration` in [@putout/plugin-remove-unreacable-code](https://github.com/coderaiser/putout/blob/v8.11.1/packages/plugin-remove-unreachable-code/lib/remove-unreachable-code.js), we can start from a test with a [fixture](https://github.com/coderaiser/putout/tree/v8.11.1/packages/plugin-remove-unreachable-code/test/fixture).

**Comment by coderaiser** [comment]
Fixed with 6986c16c583baa27ef484c861993bdf62134bc06, landed in `@putout/plugin-remove-unreachable-code v1.1.0` :tada:. Please reinstall `putout`.

Is it works for you?

**Comment by esvit** [comment]
Thanks a lot, now it works
